dashboard/app/views/pd/application/teacher1819_application_mailer/principal_approval_received.html.haml

```diff
@@ -0,0 +1,24 @@
+%p
+  Hello
+  = @application.first_name + ','
+
+%p
+  We have received your principal's approval form for your application to join Code.org's
+  Professional Learning Program. Your application is now considered complete,
+  so we will review it over the next few weeks.
+
+%p
+  Please look for a status update from us by mid-April, 2018. In the meantime,
+  don’t forget to check some
```

Comment:
small thing, but this sentence sounds funny...should this be "check out", or "check out some of these"?

Reply:
@tanyaparker preference? I like "check out some of these"

Reply:
"check out some of these" works for me